javalanglib/src/main/scala/java/lang/Math.scala

```diff
@@ -46,8 +49,19 @@ object Math {
 
   @inline def exp(a: scala.Double): scala.Double = js.Math.exp(a)
   @inline def log(a: scala.Double): scala.Double = js.Math.log(a)
-  @inline def log10(a: scala.Double): scala.Double = log(a) / 2.302585092994046
-  @inline def log1p(a: scala.Double): scala.Double = log(a + 1)
+  @inline def log10(a: scala.Double): scala.Double = {
```

Comment:
Add a blank line above this line, now that this method is not a one-liner anymore